lib/core/presentation/pages/create_post/create_post_page.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:auto_route/annotations.dart';
+import 'package:flutter/material.dart';
+
+import '../../../../i18n/i18n.g.dart';
+
+@RoutePage()
+class CreatePostPage extends StatelessWidget {
+  const CreatePostPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final textCtrl = TextEditingController();
+    final t = Translations.of(context);
+    return Scaffold(
+      backgroundColor: LemonColor.black,
+      appBar: AppBar(
+        automaticallyImplyLeading: true,
+        actions: [
+          InkWell(
+            onTap: textCtrl.text.isEmpty ? null : () {},
+            child: Row(
+              children: [
+                Container(
+                  padding: EdgeInsets.symmetric(
+                    horizontal: Spacing.smMedium,
+                    vertical: Spacing.superExtraSmall,
+                  ),
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.circular(32),
+                    color: LemonColor.lavender,
+                  ),
+                  child: Text(
+                    'Post',
+                    style: TextStyle(
+                      fontSize: 14,
+                      fontWeight: FontWeight.w700,
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+          SizedBox(width: Spacing.smMedium),
+        ],
+        elevation: 0,
+      ),
+      body: SafeArea(
+        child: Column(
+          children: [
+            Divider(color: LemonColor.white.withOpacity(0.7)),
+            Expanded(
+              child: Padding(
+                padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
+                child: TextFormField(
+                  controller: textCtrl,
+                  maxLines: 10,
+                  cursorColor: LemonColor.white,
+                  decoration: InputDecoration.collapsed(hintText: t.home.whatOnYourMind),
+                ),
+              ),
+            ),
+            Divider(color: LemonColor.white.withOpacity(0.7)),
+            Padding(
+              padding: EdgeInsets.symmetric(horizontal: Spacing.smMedium),
+              child: Row(
+                children: [
+                  Assets.icons.icHouseParty.svg(color: LemonColor.white.withOpacity(0.7)),
```

Comment:
`color` property for svg is deprecated. 
may use `ThemeSvgIcon(color: colorScheme.someColor, builder: (filter) => svg(colorFilter: filter)))`